src/ibun.cpp

```diff
@@ -128,6 +128,9 @@ usage() {
         " ",
         "Note the use of -C option with the tar command which will tar the",
         "content of mydir but without including the directory mydir in the paths.",
+        "Also note the use of the -h option with the tar command. This instructs ",
+        "tar to follow all symlinks. This means that the structured files fed to ",
+        "ibun should NOT contain symbolic links. These are not supported by ibun.",
```

Comment:
"links. These are not" -> "links as they are not"

Reply:
Done